CMakeLists.txt

```diff
@@ -178,6 +178,48 @@ if(NOT DEFINED BUILD_SHARED_LIBS)
 	endif()
 endif()
 
+option(BUILD_FUZZERS "Use BUILD_FUZZERS to build fuzzing tests" OFF)
+if(BUILD_FUZZERS)
+	set(BUILD_TESTING ON)
+
+	# A special target with fuzzer and sanitizer flags.
+	add_library(fuzzer_config INTERFACE)
+
+	target_compile_options(
```

Comment:
this stuff should be part of an external configuration, not in the main `CMakeLists.txt`

Reply:
@akallabeth What do you mean for "external configuration"? Create something like `cmake/Fuzzing.cmake` and put configuration there?